app/src/main/java/com/dkanada/gramophone/fragments/player/card/CardPlayerFragment.java

```diff
@@ -292,7 +292,8 @@ public void onToolbarToggled() {
 
     @Override
     public void onPanelSlide(View view, float slide) {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP
+                && !(Float.isInfinite(slide) || Float.isNaN(slide))) {
```

Comment:
something like this might be easier to read

```
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP && !Float.isInfinite(slide) && !Float.isNaN(slide))
```

Reply:
I've formatted the code as specified.